Cargo.toml

```diff
@@ -25,6 +25,7 @@ wayland = ["wayland-client", "wayland-protocols", "wayland-backend"]
 async-std = {version = "1.12", optional = true}
 enumflags2 = "0.7"
 futures-channel = "0.3"
+futures-lite = "1.13.0"
```

Comment:
Nothing in the commit message documents the reasoning behind this switch

Reply:
The reason is that futures-lites `filter_map` takes a simple FnMut that maps to an Option but the one from futures-utils ([docs](https://docs.rs/futures-util/latest/futures_util/stream/trait.StreamExt.html#method.filter_map)) takes an FnMut that maps to a future, resolving to the output type. That makes the implementation harder, and even when I made it compile, when I actually tried to use the resulting streams next method, the compiler complained that the steam wasn't pin. (I believe it would require to the stream)

I can include the reasoning in a commit message.

Reply:
Note that throughout the codebase two different versions of `SteamExt` will be used. I'm open to suggestions to clean that up. But not sure which would be the best way to go.

Reply:
No it's also used by `signal()` in `proxy.rs` for the type conversion.

Not sure what you mean by better semantic. Do you mean the `impl Stream` part?

Reply:
I don't think that you need futures_lite for filter_map ,see https://github.com/bilelmoussaoui/oo7/pull/40 where i used it without depending on that new dependency.

Reply:
You are totally right, I don't need futures_lite. I just realized, that futures_utils' version of `filter_map` is more general, because you can easily convert a value to a future using `ready`. I feel a little stupid that I understood it only now. I was trying to make an async block with all the logic and moving a string holding the interface name into the created future. That complicated matters a lot.
Sidenote: futures_lite is already in the dependecy tree trough zbus itself and async_std. (Of course I understand that you want to minimize direct dependencies though)

Reply:
Right, people could complain to zbus/asynd-std if they want. Thanks for the fixes!